Kuzzle/API/Controllers/AdminController.cs

```diff
@@ -0,0 +1,163 @@
+﻿using System;
+using System.Threading.Tasks;
+using KuzzleSdk.Utils;
+using Newtonsoft.Json.Linq;
+
+namespace KuzzleSdk.API.Controllers {
+  public class AdminController : BaseController {
+    internal AdminController(IKuzzleApi k) : base(k) { }
+
+
+    /// <summary>
+    /// Asynchronously create a snapshot of Kuzzle's state.
+    /// Depending on the configuration of Kuzzle.
+    /// </summary>
+    public async Task<bool> DumpAsync() {
+
+      Response response = await api.QueryAsync(new JObject {
+        {"controller", "admin"},
+        {"action", "dump"}
+      });
+
+      return (bool)response.Result["acknowledge"];
+    }
+
+    /// <summary>
+    /// Load fixtures into the storage layer.
+    /// </summary>
+    public async Task<bool> LoadFixturesAsync(JObject indexName, bool waitForRefresh = false) {
+
+      JObject query = new JObject {
+        {"controller", "admin"},
+        {"action", "loadFixtures"},
+        {"body", new JObject {
+            {"index-name", indexName}
+          }
+        }
+      };
+
+      QueryUtils.HandleRefreshOption(query, waitForRefresh);
```

Comment:
Was a good idea to create an helper but in other controllers `wait_for` is handled like this
```cs
    private void HandleRefreshOption(JObject query, bool waitForRefresh) {
      if (waitForRefresh) {
        query.Add("refresh", "wait_for");
      }
    }
```